controller/volume_controller_test.go

```diff
@@ -1173,7 +1173,7 @@ func newDaemonPod(phase corev1.PodPhase, name, namespace, nodeID, podIP string,
 			NodeName: nodeID,
 			Containers: []corev1.Container{
 				{
-					Name:  "test-container",
+					Name:  "longhorn-manager",
```

Comment:
Does this mean the test did not work before? but why we did not notice any failed cases in PR checks?

Reply:
The test worked before because we always got the first container and did not check the name.
Because I modified the implementation to get the container named "longhorn-manager", it will fail if testing with other names.